apps/google-analytics-4/lambda/src/app.ts

```diff
@@ -8,21 +9,46 @@ import {
   GoogleApiServerError,
 } from './services/google-api';
 import { ServiceAccountKeyFile } from './types';
+import { verifySignedRequestMiddleware } from './lib/verify-signed-request-middleware';
+import { InvalidSignature } from './lib/errors/invalid-signature';
+import { UnableToVerifyRequest } from './lib/errors/unable-to-verify-request';
+
+dotenv.config(); // TODO: load env vars from .env.local
+
 const app = express();
 
+// allow all OPTIONS requests
+app.options('/*', function (req, res, _next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
+  res.header(
+    'Access-Control-Allow-Headers',
+    'Content-Type, Authorization, Content-Length, X-Requested-With'
+  );
+  res.send(200);
+});
+// validate signed requests
+app.use(['/api/credentials', '/api/account_summaries'], verifySignedRequestMiddleware);
```

Comment:
Is wildcard syntax allowed here? e.g. `/api/*`?

Reply:
it's documented so, but I believe we couldn't get it to work and punted on it. i can test once more tomorrow before i merge if you'd like